embedding/embedlite/EmbedLiteView.h

```diff
@@ -125,6 +126,9 @@ class EmbedLiteView
   //   Setup renderable GL/EGL window surface size
   virtual void SetGLViewPortSize(int width, int height);
 
+  // Set screen rotation and GL world transform offset and simple rotation are allowed (orientation change)
+  virtual void SetScreenRotation(mozilla::ScreenRotation rotation, gfxMatrix matrix);
```

Comment:
Tested that as well and we need to apply both transformation and rotation. Previously with quickmozview qquickitem handled rotation as per parent chain. With plain qobject we need to do that manipulation by ourselves. 

AFAIS Compositor::ClipRectInLayersCoordinates is the crucial piece. With or without clip rect being set to the LayerManagerCompositor root, LayerManagerComposite::Render() calculates a clip rect. Transformation is need so that view is correctly rotated and screen rotation needs to match the transformation so that compositor clips correctly.